### ISSUE TITLE
GRM_GuildSync - attempt to get length of field '?'

### ISSUE THREAD

**Issue by Duranom** [TheGeneticsGuy/Guild-Roster-Manager---Group-Info#6]
Game: Wotlk Classic

Description:
The following nil error can occur when someone is logging off or on.
Cleaned all cached information, and clean install.

```Lua
Message: ...erface\AddOns\Guild_Roster_Manager\GRM_GuildSync.lua:5052: attempt to get length of field '?' (a nil value)
Time: Fri Oct 21 16:08:02 2022
Count: 2
Stack: ...erface\AddOns\Guild_Roster_Manager\GRM_GuildSync.lua:5052: attempt to get length of field '?' (a nil value)
[string "@Interface\AddOns\Guild_Roster_Manager\GRM_GuildSync.lua"]:5052: in function `CompareAltLists'
[string "@Interface\AddOns\Guild_Roster_Manager\GRM_GuildSync.lua"]:5250: in function `PreCheckChanges'
[string "@Interface\AddOns\Guild_Roster_Manager\GRM_GuildSync.lua"]:5801: in function <...erface\AddOns\Guild_Roster_Manager\GRM_GuildSync.lua:5485>

Locals: leaderListOfAlts = <table> {
}
syncRankFilter = 9
guildData = <table> {
 1 = <table> {
 }
 2 = <table> {
 }
 3 = <table> {
 }
 4 = <table> {
 }
 5 = <table> {
 }
 6 = <table> {
 }
 7 = <table> {
 }
....
// Cut off as it kept going
```

**Comment by TheGeneticsGuy** [comment]
It looks like this was for GRM core addon, not the groupInfo - Can you confirm this is for latest GRM build 1.933? /grm version

This would cause sync to fail.

**Comment by TheGeneticsGuy** [comment]
Regardless, I think I have a fix for that pushed in my new build I am about to release. Going to close this. Please request to reopen if my release doesn't fix it - it will come out 1.934 - Probably 10/21 evening.
